### PR TITLE
Fixes issue #1021

### LINKED ISSUE #1021: ✨[Feature Request]: Sending confirmation email upon Subscribing
### Is there an existing issue for this?

- [X] I have searched the existing issues

### Feature Description

User will recieve a confirmation email upon successfully subscribing to the website. And giving a alert upon subscribing.

### Use Case

Can be accessed securely.

### Benefits

_No response_

### Add ScreenShots

_No response_

### Priority

High

### Record

- [X] I have read the Contributing Guidelines
- [X] I'm a GSSOC'24 contributor
- [X] I have starred the repository

### PR DESCRIPTION
**Pull request by Bhattu-Anitha-07**
## Description

Sending confirmation email upon subscribing to the newsletter using EmailJS.

- Closes #1021 

## Type of PR

- [X] Bug fix
- [X] Feature enhancement

## Screenshots / videos (if applicable)
[c62c59cb-84c1-4df9-b67e-7360b0c2d962.webm](https://github.com/user-attachments/assets/c6c1e257-0776-444c-ae25-b5922e486ac1)

image

## Checklist

- [X] I have gone through the [contributing guide](https://github.com/Anishkagupta04/RAPIDOC-HEALTHCARE-WEBSITE-/)
- [X] I have updated my branch and synced it with project `main` branch before making this PR
- [X] I have performed a self-review of my code
- [X] I have tested the changes thoroughly before submitting this pull request.
- [X] I have provided relevant issue numbers, screenshots, and videos after making the changes.
- [X] I have commented my code, particularly in hard-to-understand areas.

### CODE REVIEW

**Comment by Bhattu-Anitha-07** [comment]
@Anishkagupta04 @varshith257 
Kindly review this PR, also assign labels.
Thank you..

**Comment by Bhattu-Anitha-07** [comment]
@varshith257 Please review this PR.

File: index.html [hunk 1]
```diff
@@ -53,6 +53,60 @@
   <link rel="stylesheet" href="./Css-Files/testimonial.css" />
   <script src="chatboxx.js" defer></script>
   <link rel="stylesheet" href="Css-Files/caduceus.css">
+  <!-- <script src="https://cdn.emailjs.com/dist/email.min.js"></script> -->
+  <script type="text/javascript"
+        src="https://cdn.jsdelivr.net/npm/@emailjs/browser@4/dist/email.min.js">
+  </script>
+    Copy code
```
Comment (varshith257) [inline index.html:60]: Can remove this line?

File: index.html [hunk 2]
```diff
@@ -905,4 +959,5 @@ <h4>RAPIDOC Newsletter</h4><br>
   <script src="js/progressbar_and_header.js"></script>
   <div id="progressBar"></div>
   <script src="js/patientportal_loginjs.js"></script>
+  <!-- <script src="functions/index.js"></script> -->
```
Comment (varshith257) [inline index.html:962]: Why this commented?

**Review by varshith257 — CHANGES_REQUESTED** [review]

**Comment by Bhattu-Anitha-07** [comment]
@varshith257 
Please add labels, Thank you.

File: index.html (hunk 1 above)
Comment (Bhattu-Anitha-07) [reply index.html:60]: Yeah but unable to commit the changes

Comment (Bhattu-Anitha-07) [reply index.html:60]: I think this line must there..
Is it giving any warnings..

File: index.html (hunk 2 above)
Comment (Bhattu-Anitha-07) [reply index.html:962]: That file is not in use.

**Comment by Bhattu-Anitha-07** [comment]
@varshith257 
Yesterday, I tried using nodemailer. However, it's not needed anymore.

File: index.html (hunk 2 above)
Comment (Bhattu-Anitha-07) [reply index.html:962]: Yesterday, I tried using nodemailer. However, it's not needed anymore.

**Comment by Bhattu-Anitha-07** [comment]
@varshith257 
Do I need to make any changes based on your request? If not, please assign labels.

**Comment by Bhattu-Anitha-07** [comment]
@varshith257 
Please let me know if any changes need to be made.

**Pull request merged**